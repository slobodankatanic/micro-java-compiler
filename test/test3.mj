program test302

int i;

{			
	void main() int j; {
		i = 2;
		j = 10;
		
		if (i == 2) {
			do {
				if (j == 5) {
					break;
				}
				if (j % 2 == 0) {
					print(j);
					print(' ');
					print('p');	
					print('a');
					print('r');
					print('a');
					print('n');
					print(eol);
				} else {
					print(j);
					print(' ');
					print('n');	
					print('e');
					print(' ');
					print('p');	
					print('a');
					print('r');
					print('a');
					print('n');
					print(eol);
				}
				j--;
			} while(j > 0);
						
			print(10);
			
		} else if (i > 2) {
			print(20);
		} else print('A');
	}
}

