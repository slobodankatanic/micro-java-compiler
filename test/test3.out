   10 paran
    9 ne paran
    8 paran
    7 ne paran
    6 paran
   10
Completion took 3 ms
